ramlfications/utils.py

```diff
@@ -117,34 +135,23 @@ def save_data(output_file, mime_types):
     with open(output_file, "w") as f:
         json.dump(mime_types, f)
 
+def setup_logger(message):
+    log = logging.getLogger(__name__)
+    log.setLevel(logging.DEBUG)
+    console = logging.StreamHandler()
+    console.setLevel(logging.DEBUG)
+    msg = "{message} - %(levelname)s - %(message)s".format(message=message)
```

Comment:
Hmm I got confused about who committed what - sorry if it wasn't you @nvillalva :)